### ISSUE TITLE
WASM Support

### ISSUE THREAD

**Issue by bevice1** [cashapp/multiplatform-paging#232]
Hey guys!

I was wondering if you are planning on releasing the library for kotlin multiplatform web with WASM as well.

kind regards

**Comment by JakeWharton** [comment]
No one is actively working on such a thing, but I'd happily accept a PR for it.

**Comment by edna-aa** [comment]
hi @JakeWharton  

here is a PR adding support https://github.com/cashapp/multiplatform-paging/pull/376

**Comment by ahna92** [comment]
hi @JakeWharton 

here is a PR adding support https://github.com/cashapp/multiplatform-paging/pull/376

**Comment by JakeWharton** [comment]
Sorry but I'm no longer in a position to spend time on this. We'll probably/hopefully completely deprecate and archive this project soon to reflect our (lack of) commitment to it.

**Comment by ahna92** [comment]
@JakeWharton oh I see

Is this because the original library will extend the platform support?

Just trying to find an alternative

**Comment by JakeWharton** [comment]
No. The AndroidX library lacks integrations with platform-specific components outside of Android views and Compose UI so it is not a suitable replacement for the capabilities here. We don't really like the paging API and are going to replace it with our own abstraction.

**Comment by wongislandd** [comment]
hello! Just wanted to put a +1 on looking for wasmjs support from a paging lib.

**Comment by chrisjenx** [comment]
I just manually copied the paging code into it's own module pointing at all the targets. I can throw into a repo if anyone is interested (fully unsupported tho)
